files/defaults.yaml

```diff
@@ -0,0 +1,277 @@
+################################################################################
```

Comment:
And what about to define templates inside yaml file and let the configmgr to do the job. This is the real templating with the only limitation of javascript. Then the combination of JCL, rexx config manager and ISREDIT macro could be omitted:

```
zowe:
  environments:
    EPLLicense: "//*\n\
      //* This program and the accompanying materials are made available\n\
      //* under the terms of the Eclipse Public License v2.0 which\n\
      //* accompanies this distribution, and is available at\n\
      //* https://www.eclipse.org/legal/epl-v20.html\n\
      //*\n\
      //* SPDX-License-Identifier: EPL-2.0\n\
      //*\n\
      //* Copyright Contributors to the Zowe Project. 2020, 2020\n\
      //*\n\
      //*********************************************************************\n"
    zweisitc: "${{ `//ZWESIS01  PROC NAME='` + components.zss.crossMemoryServerName + `',MEM=` + zowe.setup.dataset.parmlibMembers.zis.substr(6,2) + `,RGN=0M\n`
      + zowe.environments.EPLLicense
      + `//*** A SAMPLE STC FOR XMS ***\n`
      + `//* Help ...\n`
      + `//ZWESIS01 EXEC PGM=ZWESIS01,REGION=&RGN,\n`
      + `//         PARM='NAME=&NAME,MEM=&MEM'\n`
      + `//STEPLIB  DD  DSNAME=` + (() => { if (zowe.setup?.dataset?.authLoadlib) { return zowe.setup.dataset.authLoadlib } else { return '{zowe.setup.dataset.authLoadlib}' } } ()) + `,\n`
      + `//             DISP=SHR\n` 
      + `//         DD  DSNAME=` + (() => { if (zowe.setup?.dataset?.authPluginLib) { return zowe.setup.dataset.authPluginLib } else { return '{zowe.setup.dataset.authPluginLib}' } } ()) + `,\n`
      + `//             DISP=SHR\n`
      + `//PARMLIB  DD  DSNAME=` + (() => { if (zowe.setup?.dataset?.parmlib) { return zowe.setup.dataset.parmlib} else { return '{zowe.setup.dataset.parmlib}' } }()) + `,\n`
      + `//             DISP=SHR\n`
      + `//SYSPRINT DD  SYSOUT=*\n` }}"
```

Reply:
can you file that as a ticket for future enhancement?
we need to debate it: one reason to move to JCL samples was to allow people to run them without any automation at all, and have complete transparency. In the current branch they can just open up the samplib, make hand edits, and submit the jobs if they wish. They do not need ZWEGENER / "zwe init generate" / any "zwe init" step if they do not want. How can we have the same ability when the templates are within the yaml?